features/uploading_features/up_tag2concepts.feature

```diff
@@ -30,4 +30,71 @@
     
 #     When I confirm the change for concept "Squirtle" and tag "test_concept 2"
 #     And I press "Confirm Upload"
-#     Then I should see "Success"
\ No newline at end of file
+#     Then I should see "Success"
+
+Feature: Upload a Tags file
+  As a user, I can upload a tag2concepts csv file
+  And I can see new data as a result of the upload
+  
+Background: A user account exists
+
```

Comment:
Can we also include tag2concepts relations pre upload? This way, we can test for deletion